legal-api/src/legal_api/models/review.py

```diff
@@ -75,3 +75,18 @@ def get_review(cls, filing_id) -> Review:
                       filter(Review.filing_id == filing_id).
                       one_or_none())
         return review
+
+    @property
+    def json(self) -> dict:
+        """Return Review as a JSON object."""
+        return {
+            'id': self.id,
+            'nrNumber': self.nr_number,
+            'identifier': self.identifier,
+            'completingParty': self.completing_party,
+            'status': self.status.name,
+            'submissionDate': self.submission_date.isoformat() if self.submission_date else None,
+            'creationDate': self.creation_date.isoformat() if self.creation_date else None,
```

Comment:
no need to check if value exist for creationDate and submissionDate, there is a always have a value (check line 45 and 48)